lib/identity-provider.js

```diff
@@ -526,10 +526,17 @@ IdentityProvider.prototype.getGraph = function (uri, callback) {
 
 // Handle POST requests on account creation
 IdentityProvider.prototype.post = function (req, res, next) {
+  const idpMode = req.app.locals.ldp.idp
```

Comment:
this will be in req.app.locals.solid soon (so req.app.locals.ldp), we should not populate req.app.locals of too many variables since this name space could also be used by other variables.

Rules of 4 dots applies it anywhere!